src/views/LostItems/LostItemsListContainer/LostItemsListContainer.js

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  get,
+  noop,
+} from 'lodash';
+import {
+  FormattedMessage,
+} from 'react-intl';
+
+import {
+  Pane,
+  PaneMenu,
+  Paneset,
+} from '@folio/stripes/components';
+import {
+  SearchAndSortQuery,
+  SearchAndSortNoResultsMessage as NoResultsMessage,
+  CollapseFilterPaneButton,
+  ExpandFilterPaneButton,
+} from '@folio/stripes/smart-components';
+
+import {
+  Filters,
+  LostItemsList,
+} from './components';
+
+class LostItemsListContainer extends React.Component {
+  static propTypes = {
+    onNeedMoreData: PropTypes.func.isRequired,
+    queryGetter: PropTypes.func,
+    querySetter: PropTypes.func,
+    resources: PropTypes.shape({
+      records: PropTypes.object,
+    }).isRequired,
+    mutator: PropTypes.shape({
+      resultOffset: PropTypes.shape({
+        replace: PropTypes.func.isRequired,
+      }),
+    }).isRequired,
+    source: PropTypes.object,
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      filterPaneIsVisible: true,
+    };
+  }
+
+  toggleFilterPane = () => {
+    this.setState(curState => ({
+      filterPaneIsVisible: !curState.filterPaneIsVisible,
+    }));
+  }
+
+  renderResultsFirstMenu(filters) {
+    const {
+      filterPaneIsVisible,
+    } = this.state;
+
+    if (filterPaneIsVisible) {
+      return null;
+    }
+
+    const filterCount = filters.string !== '' ? filters.string.split(',').length : 0;
+
+    return (
+      <PaneMenu>
+        <ExpandFilterPaneButton
+          filterCount={filterCount}
+          onClick={this.toggleFilterPane}
+        />
+      </PaneMenu>
+    );
+  }
+
+  render() {
+    const {
+      querySetter,
+      queryGetter,
+      source,
+      onNeedMoreData,
+      resources,
+      mutator: {
+        resultOffset,
+      },
+    } = this.props;
+    const {
+      filterPaneIsVisible,
+    } = this.state;
+
+    const actualCostRecords = get(resources, 'records.records', []);
```

Comment:
@Dmitriy-Litvinenko same here. The `get` from lodash is not required with optional chaining

Reply:
Fixed